src/services/contact.js

```diff
@@ -1,11 +1,14 @@
+// Emailjs imports
 import emailjs from '@emailjs/browser';
 
+// function for sending emails that takes form event and form input values as arguments
+//check the template to know what the inputs name must be because they matter
 const mailSender = (e, templateParams) => {
-    e.preventDefault();
+    e.preventDefault(); //to stop the form from reloading the page onSubmit
 
-    emailjs.sendForm('service_iwqsu3f', 'template_d0s4lc6', templateParams, 'SMrzCAgEF5PcNX7Jo')
+    emailjs.sendForm('service_iwqsu3f', 'template_d0s4lc6', templateParams, 'SMrzCAgEF5PcNX7Jo') //emailjs method to send the form that takes the form inputs as a third argument
 
-    e.target.reset()
+    e.target.reset() //reset the form after the submit
 };
 
 export { mailSender }
```

Comment:
Please structure your comments like this:

`//comment`
`...actual code`

Reply:
you want me to write the word comment on a separate line before each comment?

Reply:
nope, move your comments one line up from the code block that it is relevant of. They should not be in the same line.